homeassistant/components/template/sensor.py

```diff
@@ -60,17 +64,20 @@
         friendly_name_template = device_config.get(CONF_FRIENDLY_NAME_TEMPLATE)
         unit_of_measurement = device_config.get(ATTR_UNIT_OF_MEASUREMENT)
         device_class = device_config.get(CONF_DEVICE_CLASS)
+        attribute_templates = device_config.get(CONF_ATTRIBUTE_TEMPLATES)
```

Comment:
Please use `dict[key]` for required config keys and keys with default config schema values.

Reply:
Done this.